software/samtools/merge/meta.yml

```diff
@@ -0,0 +1,42 @@
+name: samtools_merge
+description: Merge BAM file
+keywords:
+    - merge
+    - bam
+tools:
+    - samtools:
+        description: |
+            SAMtools is a set of utilities for interacting with and post-processing
+            short DNA sequence read alignments in the SAM, BAM and CRAM formats, written by Heng Li.
+            These files are generated as output by short read aligners like BWA.
+        homepage: http://www.htslib.org/
+        documentation: hhttp://www.htslib.org/doc/samtools.html
+        doi: 10.1093/bioinformatics/btp352
+input:
+    - meta:
+        type: map
+        description: |
+            Groovy Map containing sample information
+            e.g. [ id:'test', single_end:false ]
+    - bam:
+        type: file
+        description: BAM file
+        pattern: "*.{bam,cram,sam}"
+output:
+    - meta:
+        type: map
+        description: |
+            Groovy Map containing sample information
+            e.g. [ id:'test', single_end:false ]
+    - merged_bam:
+        type: file
+        description: BAM file
+        pattern: "*.{bam}"
+    - version:
+        type: file
+        description: File containing software version
+        pattern: "*.{version.txt}"
+authors:
+    - "@drpatelh"
+    - "@ewels"
```

Comment:
I have no idea who worked on that, so I actually copied from samtools/index and added myself

Reply:
In case it helps, seems that in the history of the main @drpatelh and @yuukiiwa  are the authors:
https://github.com/nf-core/modules/blob/master/software/samtools/merge/main.nf